cli/src/cluster_query.rs

```diff
@@ -617,8 +621,26 @@ pub fn process_show_validators(rpc_client: &RpcClient, use_lamports_unit: bool)
                 format!("{}", v)
             }
         }
+
+        fn uptime(epoch_credits: Vec<(Epoch, u64, u64)>, epoch_schedule: &EpochSchedule) -> String {
```

Comment:
The semi-duplication with `process_uptime()` doesn't feel great to me.   Think we can turn this into a common function used in both places.  That the uptime subcommand is in `vote.rs` might make this a little awkward, but the uptime subcommand could just as easily live in `cluster_query.rs`

Reply:
Sounds good, added a method here